src/components/search.js

```diff
@@ -65,25 +59,20 @@ class Search extends Component {
         `}
         render={data => {
           return (
-            <div>
-              <TextField
-                id="input-with-icon-textfield"
-                placeholder="Find apps by name, category,.."
+            <Paper className={classes.paper}>
+              <IconButton className={classes.iconButton} aria-label="Menu">
+                <Apps className={classes.AppsIcon} />
+              </IconButton>
+              <InputBase
+                placeholder="Search…"
```

Comment:
placeholder is `Find apps by name, category,..`

Reply:
done